lib/render/RayCaster.cpp

```diff
@@ -915,9 +950,10 @@ void RayCaster::_load3rdPassUniforms( int                castingMode,
         shader->SetUniformArray("lightingCoeffs", 4, coeffsF);
     }
 
-    // Pack in fast mode, lighting, and missing value booleans together
-    int flags[3] = { int(fast), int(lighting), int(_userCoordinates.missingValueMask != nullptr) };
-    shader->SetUniformArray("flags", 3, flags);
+    // Pack four booleans together
+    int flags[4] = { int(fast), int(lighting), int(insideVolume),
```

Comment:
I agree it's clearer, but my intention was to issue one data transfer command, instead of issue four, for performance considerations. 
Do you think it'd address your concern if I leave a detailed comment?

Reply:
This is negligible performance-wise

> Programmers waste enormous amounts of time thinking about, or worrying about, the speed of noncritical parts of their programs, and these attempts at efficiency actually have a strong negative impact when debugging and maintenance are considered. We should forget about small efficiencies, say about 97% of the time: premature optimization is the root of all evil. Yet we should not pass up our opportunities in that critical 3%.
> -- Donald Knuth

Reply:
Agreed. Let's not obfuscate the code for the sake of negligible performance gains.